### PR TITLE
Add documentation for cross-namespace snapshots

### PR DESCRIPTION
**Pull request by coulof**
fixes : #561

### CODE REVIEW

**Comment by ttakahashi21** [comment]
I checked the document.  Nothing from me. Thank you for asking.

File: book/src/cross-namespace-data-sources.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Cross-namespace storage data sources
+
+## Status
+
+Status | Min K8s Version | Max K8s Version | external-provisioner Version
+-------|-----------------|-----------------|-----------------------------
+Alpha  | 1.26            | -               | 3.4.0
```
Comment (xing-yang) [inline book/src/cross-namespace-data-sources.md:7]: Can you change "3.4.0" to "3.4.0+"?

Comment (coulof) [reply book/src/cross-namespace-data-sources.md]: Done

**Pull request merged**